src/node.cc

```diff
@@ -4425,6 +4425,11 @@ void FreeEnvironment(Environment* env) {
 }
 
 
+MultiIsolatePlatform* GetNodeMultiIsolatePlatform() {
```

Comment:
Redundant name, it's already in the `node::` namespace.  Just `GetMultiIsolatePlatform()` or `GetMainThreadMultiIsolatePlatform()`?

Reply:
change it to GetMainThreadMultiIsolatePlatform as a more accurate name.